packages/use-media/src/useMedia.ts

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useLayoutEffect, useState } from 'react'
+import { Effect, MediaQueryObject } from './types'
+import { noop, queryObjectToString } from './utilities'
+
+export const mockMediaQueryList: MediaQueryList = {
+  addEventListener: noop,
```

Comment:
Could we pass undefined instead of noop ?

Reply:
The `MediaQueryList` type doesn't allow `undefined`, so we'll have to stick with a function